content/docs/studio/user-guide/projects-and-experiments/live-metrics-and-plots.md

```diff
@@ -62,7 +60,7 @@ job:
     `example-get-started` repository in the `iterative` namespace,
     `STUDIO_REPO_URL` should be set to the following value:
 
-    - If you are using GitHub.com, GitLab.com or Bitbucket.org, set it to
+    - If you are using GitHub, GitLab or Bitbucket, set it to
```

Comment:
Q: is there a reason we specify these domains in a few places throughout Studio docs?

Reply:
Yes. To distinguish them from on-premises Git hosting.

Reply:
I meant GitHub vs. GitHub.com, for example.

Reply:
GitHub could also mean self-hosted server. From the [GItHub docs](https://docs.github.com/en/enterprise-server@3.5/admin/overview/about-github-enterprise-server), "GitHub Enterprise Server is a self-hosted platform for software development within your enterprise."

Reply:
OK it's a minor thing but I would recommend to be consistent everywhere. I would presonally prefer just the names (no need for .com) but up to you 🙂